docs/about.ftd

```diff
@@ -0,0 +1,172 @@
+-- $site-doc.copyright-info: Copyright © 2023 - [Gangaimaging.com](https://www.gangaimaging.com/)
+
+-- site-doc.doc-home: about
+github-url: https://github.com/fastn-community/gangaimaging.com
+color-scheme: [fastn-community.github.io/blue-heal-cs](https://fastn-community.github.io/blue-heal-cs)
+typography: [fastn-community.github.io/lato-typography](https://fastn-community.github.io/lato-typography)
+site-title: About
+
+-- site-doc.component-doc: `about`
+package-link: https://fifthtry.github.io/gangaimaging.com/
+github-link: https://github.com/fastn-community/gangaimaging.com
+package-name: fifthtry.github.io/gangaimaging.com
+info: `-- ds.about` can be used as a banner on gangaimaging.com pages.
+
+`-- ds.about` contains, title, body, sub-titles and images. If you want to use
```

Comment:
```suggestion
`-- ds.about` contains, title, body, sub-titles and images. If you want to use
```
@MeenuKumari28 while navigating to this page, left nav shrinks, because : 
```css
max-width: 1320px;
padding-bottom: 64px;
width: 1140px;
```
Fix this, the left navbar should be always visible with same width.